static/app/actionCreators/indicator.tsx

```diff
@@ -83,7 +83,15 @@ export function addLoadingMessage(
 }
 
 export function addErrorMessage(msg: React.ReactNode, options?: Options) {
-  return addMessageWithType('error')(msg, options);
+  if (typeof msg === 'string' || React.isValidElement(msg)) {
+    return addMessageWithType('error')(msg, options);
+  }
+  return addMessageWithType('error')(
+    t(
+      "It looks like you've hit an issue in our server API. It's likely we're already looking into this!"
```

Comment:
Hmmm does this fit?
```suggestion
      "You've hit an issue, fortunately we use Sentry to monitor Sentry. So It's likely we're already looking into this!"
```

Reply:
yeah 👍 something along those lines sounds good

Reply:
<img width="896" alt="Screen Shot 2022-04-07 at 3 41 09 PM" src="https://user-images.githubusercontent.com/63818634/162283489-d1b1490b-3c77-4f9c-89a8-7631715aad3b.png">